### PR TITLE
[HDF5Application] Update README.md

### PR DESCRIPTION
**Pull request by loumalouomega**
**Description**
The description in the README.md is not correct

**Changelog**
- Correct instruction

### CODE REVIEW

File: applications/HDF5Application/README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ The HDF5 C libraries are used with the *HDF5Application*. If Kratos is configure
 2. Configure Kratos to build the *HDF5Application*
 
     ```
-    -DHDF5_APPLICATION=ON \
+    add_app ${KRATOS_APP_DIR}/HDF5Application
```
Comment (philbucher) [inline applications/HDF5Application/README.md]: this won't work for win, probably better to point to the corresponding section of INSTALL.md
https://github.com/KratosMultiphysics/Kratos/blob/master/INSTALL.md#adding-applications

**Review by philbucher — COMMENTED** [review]
thx, minor comment

File: applications/HDF5Application/README.md (hunk 1 above)
Comment (loumalouomega) [reply applications/HDF5Application/README.md]: True

Comment (loumalouomega) [reply applications/HDF5Application/README.md]: Done

**Review by philbucher — APPROVED** [review]
I would have left the actual call out completely as there is still e.g. MacOS missing and if it changes in the future it will be outdated again but ok for me

**Comment by philbucher** [comment]
Thanks

**Pull request merged**